apps/accounts/views/oauth2_profile.py

```diff
@@ -37,7 +40,9 @@ def get_userinfo(user):
 @protected_resource()
 def openidconnect_userinfo(request, **kwargs):
     if request.path.startswith('/v2') and (not waffle.flag_is_active(request, 'bfd_v2_flag')):
-        raise exceptions.NotFound("bfd_v2_flag not active.")
+        err = exceptions.NotFound("bfd_v2_flag not active.")
+        log_v2_blocked(request.user, request.auth.application, err, fhir='CapabilityStatement')
```

Comment:
This is a bit confusing since OIDC userinfo endpoint isn't a FHIR capability statement.  Can we put something else there, or just dump the request path, etc?